slo-monitor/slo-monitor-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: slo-monitor
+  labels:
+    app: slo-monitor
+spec:
+  selector:
+    app: slo-monitor
+  ports:
+  - name: metrics
+    port: 80
+    targetPort: metrics
+  type: LoadBalancer
```

Comment:
It's useful when running as a Deployment, and LoadBalancer type makes it easier to reach from outside the cluster.